sdk/containerregistry/container-registry/package.json

```diff
@@ -33,7 +33,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "generate:client": "autorest --typescript --v3 swagger",
-    "integration-test:browser": "karma start --single-run",
+    "integration-test:browser": "echo skipped",
```

Comment:
This package doesn't run in browsers?

Reply:
Currently due to Linux agent not having the required version to run script to import images, I am using the ACR management library for this release.  It uses `@azure/ms-rest-nodeauth` which doesn't run on browser.  I have this issue tracking re-enabling browser tests: https://github.com/azure/azure-sdk-for-js/issues/14697